### DISCUSSION TITLE
`TREXIO` group indirection in HDF5 backend

### DISCUSSION THREAD

**Discussion by wavefunction91** [TREX-CoE/trexio#152] [Ideas]
It may be useful to further namespace `trexio` data in the case of shared HDF5 files, e.g. `/nucleus` -> `/trexio/nucleus`, etc. This may be not required in the event that #117 gets merged.

**Comment by q-posev** [comment]
I thought HDF5 had a functionality to link HDF5 files. It seems cleaner than having one HDF5 file to rule them all. What am I missing?
